src/scripts/cfo.zig

```diff
@@ -0,0 +1,484 @@
+//! Continuous Fuzzing Orchestrator.
+//!
+//! We have a number of machines which run
+//!
+//!     git clone https://github.com/tigerbeetle/tigerbeetle && cd tigerbeetle
+//!     while True:
+//!         git fetch origin && git reset --hard origin/main
+//!         ./scripts/install_zig.sh
+//!         ./zig/zig build scripts -- cfo
+//!
+//! By modifying this script, we can make those machines do interesting things.
+//!
+//! The primary use-case is fuzzing: `cfo` runs a random fuzzer, and, if it finds a failure, it is
+//! recorded in devhubdb.
+//!
+//! Specifically:
+//!
+//! CFO keeps `args.concurrency` fuzzes running at the same time. For simplicity, it polls currently
+//! running fuzzers for completion every second in a fuzzing loop. A fuzzer fails if it returns
+//! non-zero error code.
+//!
+//! The fuzzing loops runs for `args.budget_minutes`. To detect hangs, if any fuzzer is still
+//! running after additional `args.hang_minutes`, it is killed (thus returning non-zero status and
```

Comment:
This is a neat way to handle both keeping the concurrency and hanging!

Reply:
Although - it might be too conservative currently. Managed to hit it on my laptop which is pretty fast.

Reply:
I’d say that cache map might be a too slow fuzzer under some configurations? I’d say half an hour per fuzzer is what we’d want here

Reply:
We can probably limit its fuzz ops quite dramatically, I would be surprised if it weren't one of the quicker fuzzers on a per op basis...